src/main/services/additional-content/maps/local-maps-manager.service.ts

```diff
@@ -334,25 +334,30 @@ export class LocalMapsManagerService {
 
             (async () => {
                 const mapsPath = await this.getMapsFolderPath(version);
-                root: for(const zipPath of zipPaths) {
-                    if(unsubscribed) { break; }
+                for(const zipPath of zipPaths) {
+
+                    if(unsubscribed) {
+                        log.info("Maps importation from zip has been cancelled");
+                        return;
+                    }
 
                     progress = { total: 0, current: 0 };
                     obs.next(progress); // reset progress for each zip
 
                     if(!pathExistsSync(zipPath)) { continue; }
+
                     const zip = new StreamZip.async({ file: zipPath });
                     const { result: zipEntries, error } = await tryit(() => zip.entries());
                     const zipEntriesValues = Object.values(zipEntries);
 
                     if(error) {
                         log.error("Could not read zip entries", zipPath, error);
-                        zip.close();
+                        await zip.close();
                         continue;
                     }
 
                     const mapsFolders = zipEntriesValues.reduce((acc, entry) => {
-                        if(!/(^|\/)(I|i)nfo.dat$/.test(entry.name)){ return acc; }
+                        if(!/(^|\/)[I-i]nfo\.dat$/.test(entry.name)){ return acc; }
```

Comment:
Shouldn't this be `(I|i)` not `[I-i]` since the latter means that any character between I and i, so `Jnfo.dat` or `Knfo.dat` would pass this test.

Reply:
Yep, you'r right 😅 I've been fooled by sonarcloud x)
I will remove the `-` 👌

Reply:
Ohh yeah that's much simpler